examples/bridge-app/bridge-common/gen/af-gen-event.h

```diff
@@ -0,0 +1,72 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *
+ *    Copyright (c) 2020 Silicon Labs
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+// This file is generated by Simplicity Studio.  Please do not edit manually.
```

Comment:
Guessing this is ember and we have to fix the comment? Or was this really simplicyt studio?

Reply:
> Marking approved - my comments were nitpicks and these are too many generated files for me to actually review (we need to figure out what to do here ... this does not seem sustainable in the long term).
> 
> For now assumption is we do want to develop bridge examples and iterate over them, so may as well get this in.

Most of the generated files are directly taken from the lighting-app example. So any of the nits which exist here would probably need to be fixed in that as well. I think we can get this in and once we figure out a sustainable way to do generated files, then migrate both applications to using that?

Reply:
For the record this file is still not generated. The meta tracker for it is #3637.